src/notebooks/pipes/Data/FieldsList.tsx

```diff
@@ -11,31 +11,29 @@ import Selectors from 'src/notebooks/pipes/Data/Selectors'
 import {SchemaContext} from 'src/notebooks/context/schemaProvider'
 
 const FieldsList: FC = () => {
-  const {loading, data} = useContext(SchemaContext)
-
-  let body = <span />
+  const {loading} = useContext(SchemaContext)
 
   if (loading === RemoteDataState.Loading) {
-    body = (
+    return (
       <div className="data-source--list__empty">
         <TechnoSpinner strokeWidth={ComponentSize.Small} diameterPixels={32} />
       </div>
     )
   }
 
   if (loading === RemoteDataState.Error) {
-    body = (
+    return (
       <div className="data-source--list__empty">
         <p>Could not fetch schema</p>
       </div>
     )
   }
 
-  if (loading === RemoteDataState.Done && Object.keys(data).length > 0) {
-    body = <Selectors />
+  if (loading === RemoteDataState.Done) {
```

Comment:
not sure how I feel about removing the data condition here.

Reply:
The reason being, what happens in the event that there isn't any data loaded?

Reply:
yeah, it's a bit of a regression not gonna lie. but the same functionality of that guard should be carried out within the `Selectors` and i didn't know exactly how to map the intent of the data field with the measurements/fields/tags concept. i'd be happy to add the guard back in if you wouldn't mind explaining that part

Reply:
makes sense